### PR TITLE
Hero component

### PR DESCRIPTION
**Pull request by Donivanes**
### Describe your changes:

- Added hero component
- Added new gradient variable and font sizes to theme
- Missing the correct font family (@robertoHeCi on it)

## Images: 

- Desktop
FireShot Capture 004 - Vite + React + TS - localhost
- Mobile
FireShot Capture 005 - Vite + React + TS - localhost

### Issue ticket number and link:

- Number : #37

### Checklist before requesting a review:

- [x] Your code builds clean without any errors or warnings
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: src/stitches.config.ts [hunk 1]
```diff
@@ -18,7 +18,9 @@ export const {
       blue500: 'hsl(206,100%,50%)',
       purple500: 'hsl(252,78%,60%)',
       green500: 'hsl(148,60%,60%)',
-      red500: 'hsl(352,100%,62%)'
+      red500: 'hsl(352,100%,62%)',
+      gradientBlueToGray: 'linear-gradient(0deg, rgba(30, 30, 30, 0.5) 0%, rgba(50, 86, 96, 0.8) 33.85%, #428293 62.5%, rgba(97, 218, 251, 0.75) 100%)',
```
Comment (jgleal) [inline src/stitches.config.ts:22]: nice!! I like this approach!

File: src/components/Hero/Hero.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  HeroImage,
+  HeroButton
+} from './Hero.styles'
+
+export const Hero: React.FC = () => {
+  return (
+    <HeroSection>
+      <HeroContent>
+        <HeroTitle>Shield.js</HeroTitle>
+        <HeroSubtitle>The new frontend project by One&nbsp;Beyond</HeroSubtitle>
```
Comment (jgleal) [inline src/components/Hero/Hero.tsx:18]: should we use i18n for whole page? or just a section like an example?

**Review by acalvom — APPROVED** [review]
Well done 🥇 
So much to learn from you 😄

File: src/components/Hero/Hero.test.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { render, screen } from '@testing-library/react'
+import { expect, test } from 'vitest'
+
+import { Hero } from './Hero'
+
+test('renders Hero component', () => {
+  render(
+    <Hero />
+  )
+  expect(screen.getByText('Shield.js')).toBeInTheDocument()
```
Comment (jgleal) [inline src/components/Hero/Hero.test.tsx:10]: @Betisman is proud of you 🎉

**Review by jgleal — APPROVED** [review]
Well done @Donivanes !! I like a lot!!! I look forward to release!!

File: src/components/Hero/Hero.tsx (hunk 2 above)
Comment (robertoHeCi) [reply src/components/Hero/Hero.tsx:18]: I would put the texts in the i18n as we have in the pages that already exist

File: src/stitches.config.ts (hunk 1 above)
Comment (robertoHeCi) [reply src/stitches.config.ts:22]: 🔝

File: src/stitches.config.ts [hunk 4]
```diff
@@ -27,8 +29,14 @@ export const {
     },
     fontSizes: {
       1: '12px',
-      2: '13px',
-      3: '15px'
+      2: '14x',
```
Comment (robertoHeCi) [inline src/stitches.config.ts:32]: There is a typo in the 14 👀 
What if we move from px to rem?

File: src/components/Hero/Hero.tsx (hunk 2 above)
Comment (Donivanes) [reply src/components/Hero/Hero.tsx:18]: I'll do it

File: src/stitches.config.ts (hunk 4 above)
Comment (Donivanes) [reply src/stitches.config.ts:32]: We can use both, I'll modify it in rem

**Comment by Donivanes** [comment]
I've done the changes that you've suggested @jgleal @robertoHeCi

**Pull request merged**